pkg/daemon/daemon.go

```diff
@@ -653,8 +668,31 @@ func (dn *Daemon) syncNode(key string) error {
 		return nil
 	}
 
+	if node.Annotations[constants.MachineConfigDaemonPostConfigAction] == "Rebooting" {
```

Comment:
question (non-blocking): Do we have a constant for `Rebooting` someplace?

Reply:
nope. I can add one